app/components/Combobox.tsx

```diff
@@ -52,6 +53,7 @@ export function Combobox<T extends Record<string, string | null | number>>({
   id,
   isLoading = false,
   fullWidth = false,
+  fuseOptions,
```

Comment:
Instead of doing

```ts
 if (typeof fuseOptions === "undefined") {
    fuseOptions = {};
  }
```

below it could be a bit cleaner to just simply do:

```suggestion
  fuseOptions = {},
```

Then if fuseOptions is nullish it gets empty object as default value.

Reply:
Makes sense, I forgot about the default initializer for JS args - thanks again!